modules/sr/robot/ruggeduino_devices.py

```diff
@@ -1,7 +1,12 @@
 import logging
 
-from controller import Robot
-from sr.robot.utils import map_to_range
+from controller import (
+    LED,
+    Robot,
+    TouchSensor,
+    DistanceSensor as TDistanceSensor,
```

Comment:
I would encourage not using `T` as the prefix here -- that tends to imply that this is a generic type parameter, which I don't think that this is. Perhaps `WebotsDistanceSensor`?

Reply:
Yeah `WebotsDistanceSensor` is a better name hopefully this will be superseded by #250